pkg/catalog/catalog.go

```diff
@@ -1230,7 +1230,7 @@ func (c *Catalog) CreateCommitRecord(ctx context.Context, repositoryID string, c
 		Committer:    committer,
 		Message:      message,
 		MetaRangeID:  graveler.MetaRangeID(metaRangeID),
-		CreationDate: time.Unix(0, *creationDate).UTC(),
+		CreationDate: time.Unix(*creationDate, 0).UTC(),
```

Comment:
If you're already here...
Can you please change the creationDate from `*int64` to `int64` ?